doc/source/cluster/kubernetes/user-guides/gpu.rst

```diff
@@ -69,6 +70,22 @@ Each of the Ray pods in the group can be scheduled on an AWS `p2.xlarge` instanc
     Finally, make sure your group or pool of GPU Kubernetes nodes are configured to autoscale.
     Refer to your :ref:`cloud provider's documentation <kuberay-k8s-setup>` for details on autoscaling node pools.
 
+GPU multi-tenancy
+_________________
+
+If a Pod doesn't include `nvidia.com/gpu` in its resource configurations, even if it's scheduled on a GPU node, users typically expect the Pod not to have visibility of any GPU devices.
+However, if `nvidia.com/gpu` isn't specified, the default value for `NVIDIA_VISIBLE_DEVICES` is `all`.
+In other words, the Pod has visibility of all GPU devices on the node.
+This behavior isn't unique to KubeRay, but rather a known issue for NVIDIA.
+A workaround is to set the `NVIDIA_VISIBLE_DEVICES` environment variable to `void` in the Pods which don't require GPU devices.
+
+Some useful links:
+
+- `NVIDIA/k8s-device-plugin#61`_
+- `NVIDIA/k8s-device-plugin#87`_
+- `[NVIDIA] Preventing unprivileged access to GPUs in Kubernetes`_
+- `ray-project/ray#29753`_
+
```

Comment:
Good to know! I wonder if cloud managed Kubernetes services (GKE, EKS) automatically implement a workaround for this.